### ISSUE TITLE
libglib2.0-0:i386 depends on zlib1g, but zlib1g is not installed

### ISSUE THREAD

**Issue by Amazeryogo** [pop-os/pop#1310]
**Distribution (run `cat /etc/os-release`):**
 NAME="Pop!_OS"
VERSION="20.04 LTS"
ID=pop
ID_LIKE="ubuntu debian"
PRETTY_NAME="Pop!_OS 20.04 LTS"
VERSION_ID="20.04"
HOME_URL="https://pop.system76.com"
SUPPORT_URL="https://support.system76.com"
BUG_REPORT_URL="https://github.com/pop-os/pop/issues"
PRIVACY_POLICY_URL="https://system76.com/privacy"
VERSION_CODENAME=focal
UBUNTU_CODENAME=focal
LOGO=distributor-logo-pop-os
 
 
**Related Application and/or Package Version (run `apt policy $PACKAGE NAME`):**

**Issue/Bug Description:**
The pop!_shop is not able to install these packages 

 gir1.2-gweather-3.0 GObject introspection data for the GWeather library Version: 3.36.1-1~ubuntu20.04.1
 libpop-upgrade-gtk The Pop upgrade experience as a GTK widget in a dylib Version: 0.1.0~1603227722~20.04~2736463
 pop-upgrade Utility for performing system upgrades on Pop!_OS Version: 0.1.0~1603227722~20.04~2736463

** when I try to upgrade it through apt I get this:**

 The following packages have unmet dependencies:
 libavcodec58:i386 : Depends: zlib1g:i386 (>= 1:1.2.0) but it is not installed
 libcairo2:i386 : Depends: zlib1g:i386 (>= 1:1.1.4) but it is not installed
 libcups2:i386 : Depends: zlib1g:i386 (>= 1:1.2.0) but it is not installed
 libcurl3-gnutls:i386 : Depends: zlib1g:i386 (>= 1:1.1.4) but it is not installed
 libelf1:i386 : Depends: zlib1g:i386 (>= 1:1.1.4) but it is not installed
 libfreetype6:i386 : Depends: zlib1g:i386 (>= 1:1.1.4) but it is not installed
 libgd3:i386 : Depends: zlib1g:i386 (>= 1:1.1.4) but it is not installed
 libgl1-mesa-dri:i386 : Depends: zlib1g:i386 (>= 1:1.1.4) but it is not installed
 libglib2.0-0:i386 : Depends: zlib1g:i386 (>= 1:1.2.2) but it is not installed
 libgstreamer-plugins-base1.0-0:i386 : Depends: zlib1g:i386 (>= 1:1.1.4) but it is not installed
 libllvm10:i386 : Depends: zlib1g:i386 (>= 1:1.2.0) but it is not installed
 libmysqlclient21:i386 : Depends: zlib1g:i386 (>= 1:1.1.4) but it is not installed
 libosmesa6:i386 : Depends: zlib1g:i386 (>= 1:1.1.4) but it is not installed
 libpci3:i386 : Depends: zlib1g:i386 (>= 1:1.1.4) but it is not installed
 libpciaccess0:i386 : Depends: zlib1g:i386 (>= 1:1.2.3.3) but it is not installed
 libperl5.30:i386 : Depends: zlib1g:i386 (>= 1:1.2.2.3) but it is not installed
 libpng16-16:i386 : Depends: zlib1g:i386 (>= 1:1.2.11) but it is not installed
 librtmp1:i386 : Depends: zlib1g:i386 (>= 1:1.1.4) but it is not installed
 libssh-4:i386 : Depends: zlib1g:i386 (>= 1:1.1.4) but it is not installed
 libtiff5:i386 : Depends: zlib1g:i386 (>= 1:1.1.4) but it is not installed
 libwine:i386 : Depends: zlib1g:i386 (>= 1:1.1.4) but it is not installed
                Recommends: libodbc1:i386 (>= 2.3.1) but it is not installed
                Recommends: libasound2-plugins:i386 but it is not installed
                Recommends: gstreamer1.0-plugins-good:i386 but it is not installed
 libxml2:i386 : Depends: zlib1g:i386 (>= 1:1.2.3.3) but it is not installed
 mesa-vdpau-drivers:i386 : Depends: zlib1g:i386 (>= 1:1.1.4) but it is not installed
E: Unmet dependencies. Try 'apt --fix-broken install' with no packages (or specify a solution).

** when I try to apt --fix-broken install **

dpkg: dependency problems prevent processing triggers for libglib2.0-0:i386:
 libglib2.0-0:i386 depends on zlib1g (>= 1:1.2.2).

dpkg: error processing package libglib2.0-0:i386 (--configure):
 dependency problems - leaving triggers unprocessed
dpkg: dependency problems prevent processing triggers for libglib2.0-0:i386:
 libglib2.0-0:i386 depends on zlib1g (>= 1:1.2.2).

dpkg: error processing package libglib2.0-0:i386 (--configure):
 dependency problems - leaving triggers unprocessed
dpkg: dependency problems prevent processing triggers for libglib2.0-0:i386:
 libglib2.0-0:i386 depends on zlib1g (>= 1:1.2.2).

dpkg: error processing package libglib2.0-0:i386 (--configure):
 dependency problems - leaving triggers unprocessed
dpkg: dependency problems prevent processing triggers for libglib2.0-0:i386:
 libglib2.0-0:i386 depends on zlib1g (>= 1:1.2.2).

dpkg: error processing package libglib2.0-0:i386 (--configure):
 dependency problems - leaving triggers unprocessed
No apport report written because MaxReports is reached already
                                                              dpkg: dependency 
problems prevent processing triggers for libglib2.0-0:i386:
 libglib2.0-0:i386 depends on zlib1g (>= 1:1.2.2).

dpkg: error processing package libglib2.0-0:i386 (--configure):
 dependency problems - leaving triggers unprocessed
No apport report written because MaxReports is reached already
                                                              dpkg: dependency 
problems prevent processing triggers for libglib2.0-0:i386:
 libglib2.0-0:i386 depends on zlib1g (>= 1:1.2.2).

dpkg: error processing package libglib2.0-0:i386 (--configure):
 dependency problems - leaving triggers unprocessed
No apport report written because MaxReports is reached already
                                                              dpkg: dependency 
problems prevent processing triggers for libglib2.0-0:i386:
 libglib2.0-0:i386 depends on zlib1g (>= 1:1.2.2).

dpkg: error processing package libglib2.0-0:i386 (--configure):
 dependency problems - leaving triggers unprocessed
No apport report written because MaxReports is reached already
                                                              dpkg: dependency 
problems prevent processing triggers for libglib2.0-0:i386:
 libglib2.0-0:i386 depends on zlib1g (>= 1:1.2.2).

dpkg: error processing package libglib2.0-0:i386 (--configure):
 dependency problems - leaving triggers unprocessed
No apport report written because MaxReports is reached already
                                                              dpkg: dependency 
problems prevent processing triggers for libglib2.0-0:i386:
 libglib2.0-0:i386 depends on zlib1g (>= 1:1.2.2).

dpkg: error processing package libglib2.0-0:i386 (--configure):
 dependency problems - leaving triggers unprocessed
No apport report written because MaxReports is reached already
                                                              dpkg: dependency 
problems prevent processing triggers for libglib2.0-0:i386:
 libglib2.0-0:i386 depends on zlib1g (>= 1:1.2.2).

dpkg: error processing package libglib2.0-0:i386 (--configure):
 dependency problems - leaving triggers unprocessed
No apport report written because MaxReports is reached already
                                                              dpkg: dependency 
problems prevent processing triggers for libglib2.0-0:i386:
 libglib2.0-0:i386 depends on zlib1g (>= 1:1.2.2).

dpkg: error processing package libglib2.0-0:i386 (--configure):
 dependency problems - leaving triggers unprocessed
No apport report written because MaxReports is reached already
                                                              dpkg: dependency 
problems prevent processing triggers for libglib2.0-0:i386:
 libglib2.0-0:i386 depends on zlib1g (>= 1:1.2.2).

dpkg: error processing package libglib2.0-0:i386 (--configure):
 dependency problems - leaving triggers unprocessed
No apport report written because MaxReports is reached already
                                                              dpkg: dependency 
problems prevent processing triggers for libglib2.0-0:i386:
 libglib2.0-0:i386 depends on zlib1g (>= 1:1.2.2).

dpkg: error processing package libglib2.0-0:i386 (--configure):
 dependency problems - leaving triggers unprocessed
No apport report written because MaxReports is reached already
                                                              dpkg: dependency 
problems prevent processing triggers for libglib2.0-0:i386:
 libglib2.0-0:i386 depends on zlib1g (>= 1:1.2.2).

dpkg: error processing package libglib2.0-0:i386 (--configure):
 dependency problems - leaving triggers unprocessed
No apport report written because MaxReports is reached already
                                                              dpkg: dependency 
problems prevent processing triggers for libglib2.0-0:i386:
 libglib2.0-0:i386 depends on zlib1g (>= 1:1.2.2).

dpkg: error processing package libglib2.0-0:i386 (--configure):
 dependency problems - leaving triggers unprocessed
No apport report written because MaxReports is reached already
                                                              dpkg: dependency 
problems prevent processing triggers for libglib2.0-0:i386:
 libglib2.0-0:i386 depends on zlib1g (>= 1:1.2.2).

dpkg: error processing package libglib2.0-0:i386 (--configure):
 dependency problems - leaving triggers unprocessed
No apport report written because MaxReports is reached already
                                                              dpkg: dependency 
problems prevent processing triggers for libglib2.0-0:i386:
 libglib2.0-0:i386 depends on zlib1g (>= 1:1.2.2).

dpkg: error processing package libglib2.0-0:i386 (--configure):
 dependency problems - leaving triggers unprocessed
No apport report written because MaxReports is reached already
                                                              dpkg: dependency 
problems prevent processing triggers for libglib2.0-0:i386:
 libglib2.0-0:i386 depends on zlib1g (>= 1:1.2.2).

dpkg: error processing package libglib2.0-0:i386 (--configure):
 dependency problems - leaving triggers unprocessed
No apport report written because MaxReports is reached already
                                                              dpkg: dependency 
problems prevent processing triggers for libglib2.0-0:i386:
 libglib2.0-0:i386 depends on zlib1g (>= 1:1.2.2).

dpkg: error processing package libglib2.0-0:i386 (--configure):
 dependency problems - leaving triggers unprocessed
No apport report written because MaxReports is reached already
                                                              dpkg: dependency 
problems prevent processing triggers for libglib2.0-0:i386:
 libglib2.0-0:i386 depends on zlib1g (>= 1:1.2.2).

dpkg: error processing package libglib2.0-0:i386 (--configure):
 dependency problems - leaving triggers unprocessed
No apport report written because MaxReports is reached already
                                                              dpkg: dependency 
problems prevent processing triggers for libglib2.0-0:i386:
 libglib2.0-0:i386 depends on zlib1g (>= 1:1.2.2).

dpkg: error processing package libglib2.0-0:i386 (--configure):
 dependency problems - leaving triggers unprocessed
No apport report written because MaxReports is reached already
                                                              dpkg: dependency 
problems prevent processing triggers for libglib2.0-0:i386:
 libglib2.0-0:i386 depends on zlib1g (>= 1:1.2.2).

dpkg: error processing package libglib2.0-0:i386 (--configure):
 dependency problems - leaving triggers unprocessed
No apport report written because MaxReports is reached already
                                                              dpkg: dependency 
problems prevent processing triggers for libglib2.0-0:i386:
 libglib2.0-0:i386 depends on zlib1g (>= 1:1.2.2).

dpkg: error processing package libglib2.0-0:i386 (--configure):
 dependency problems - leaving triggers unprocessed
No apport report written because MaxReports is reached already
                                                              dpkg: dependency 
problems prevent processing triggers for libglib2.0-0:i386:
 libglib2.0-0:i386 depends on zlib1g (>= 1:1.2.2).

dpkg: error processing package libglib2.0-0:i386 (--configure):
 dependency problems - leaving triggers unprocessed
No apport report written because MaxReports is reached already
                                                              dpkg: dependency 
problems prevent processing triggers for libglib2.0-0:i386:
 libglib2.0-0:i386 depends on zlib1g (>= 1:1.2.2).

dpkg: error processing package libglib2.0-0:i386 (--configure):
 dependency problems - leaving triggers unprocessed
No apport report written because MaxReports is reached already
                                                              dpkg: dependency 
problems prevent processing triggers for libglib2.0-0:i386:
 libglib2.0-0:i386 depends on zlib1g (>= 1:1.2.2).

dpkg: error processing package libglib2.0-0:i386 (--configure):
 dependency problems - leaving triggers unprocessed
No apport report written because MaxReports is reached already
                                                              dpkg: dependency 
problems prevent processing triggers for libglib2.0-0:i386:
 libglib2.0-0:i386 depends on zlib1g (>= 1:1.2.2).

dpkg: error processing package libglib2.0-0:i386 (--configure):
 dependency problems - leaving triggers unprocessed
No apport report written because MaxReports is reached already
                                                              dpkg: dependency 
problems prevent processing triggers for libglib2.0-0:i386:
 libglib2.0-0:i386 depends on zlib1g (>= 1:1.2.2).

dpkg: error processing package libglib2.0-0:i386 (--configure):
 dependency problems - leaving triggers unprocessed
No apport report written because MaxReports is reached already
                                                              dpkg: dependency 
problems prevent processing triggers for libglib2.0-0:i386:
 libglib2.0-0:i386 depends on zlib1g (>= 1:1.2.2).

dpkg: error processing package libglib2.0-0:i386 (--configure):
 dependency problems - leaving triggers unprocessed
No apport report written because MaxReports is reached already
                                                              dpkg: dependency 
problems prevent processing triggers for libglib2.0-0:i386:
 libglib2.0-0:i386 depends on zlib1g (>= 1:1.2.2).

dpkg: error processing package libglib2.0-0:i386 (--configure):
 dependency problems - leaving triggers unprocessed
No apport report written because MaxReports is reached already
                                                              dpkg: dependency 
problems prevent processing triggers for libglib2.0-0:i386:
 libglib2.0-0:i386 depends on zlib1g (>= 1:1.2.2).

dpkg: error processing package libglib2.0-0:i386 (--configure):
 dependency problems - leaving triggers unprocessed
No apport report written because MaxReports is reached already
                                                              dpkg: dependency 
problems prevent processing triggers for libglib2.0-0:i386:
 libglib2.0-0:i386 depends on zlib1g (>= 1:1.2.2).

dpkg: error processing package libglib2.0-0:i386 (--configure):
 dependency problems - leaving triggers unprocessed
No apport report written because MaxReports is reached already
                                                              dpkg: dependency 
problems prevent processing triggers for libglib2.0-0:i386:
 libglib2.0-0:i386 depends on zlib1g (>= 1:1.2.2).

dpkg: error processing package libglib2.0-0:i386 (--configure):
 dependency problems - leaving triggers unprocessed
No apport report written because MaxReports is reached already
                                                              dpkg: dependency 
problems prevent processing triggers for libglib2.0-0:i386:
 libglib2.0-0:i386 depends on zlib1g (>= 1:1.2.2).

dpkg: error processing package libglib2.0-0:i386 (--configure):
 dependency problems - leaving triggers unprocessed
No apport report written because MaxReports is reached already
                                                              dpkg: dependency 
problems prevent processing triggers for libglib2.0-0:i386:
 libglib2.0-0:i386 depends on zlib1g (>= 1:1.2.2).

dpkg: error processing package libglib2.0-0:i386 (--configure):
 dependency problems - leaving triggers unprocessed
No apport report written because MaxReports is reached already
                                                              dpkg: dependency 
problems prevent processing triggers for libglib2.0-0:i386:
 libglib2.0-0:i386 depends on zlib1g (>= 1:1.2.2).

dpkg: error processing package libglib2.0-0:i386 (--configure):
 dependency problems - leaving triggers unprocessed
No apport report written because MaxReports is reached already
                                                              dpkg: dependency 
problems prevent processing triggers for libglib2.0-0:i386:
 libglib2.0-0:i386 depends on zlib1g (>= 1:1.2.2).

dpkg: error processing package libglib2.0-0:i386 (--configure):
 dependency problems - leaving triggers unprocessed
No apport report written because MaxReports is reached already
                                                              dpkg: dependency 
problems prevent processing triggers for libglib2.0-0:i386:
 libglib2.0-0:i386 depends on zlib1g (>= 1:1.2.2).

dpkg: error processing package libglib2.0-0:i386 (--configure):
 dependency problems - leaving triggers unprocessed
No apport report written because MaxReports is reached already
                                                              dpkg: dependency 
problems prevent processing triggers for libglib2.0-0:i386:
 libglib2.0-0:i386 depends on zlib1g (>= 1:1.2.2).

dpkg: error processing package libglib2.0-0:i386 (--configure):
 dependency problems - leaving triggers unprocessed
No apport report written because MaxReports is reached already
                                                              dpkg: dependency 
problems prevent processing triggers for libglib2.0-0:i386:
 libglib2.0-0:i386 depends on zlib1g (>= 1:1.2.2).

dpkg: error processing package libglib2.0-0:i386 (--configure):
 dependency problems - leaving triggers unprocessed
No apport report written because MaxReports is reached already
                                                              dpkg: dependency 
problems prevent processing triggers for libglib2.0-0:i386:
 libglib2.0-0:i386 depends on zlib1g (>= 1:1.2.2).

dpkg: error processing package libglib2.0-0:i386 (--configure):
 dependency problems - leaving triggers unprocessed
No apport report written because MaxReports is reached already
                                                              dpkg: dependency 
problems prevent processing triggers for libglib2.0-0:i386:
 libglib2.0-0:i386 depends on zlib1g (>= 1:1.2.2).

dpkg: error processing package libglib2.0-0:i386 (--configure):
 dependency problems - leaving triggers unprocessed
No apport report written because MaxReports is reached already
                                                              dpkg: dependency 
problems prevent processing triggers for libglib2.0-0:i386:
 libglib2.0-0:i386 depends on zlib1g (>= 1:1.2.2).

dpkg: error processing package libglib2.0-0:i386 (--configure):
 dependency problems - leaving triggers unprocessed
No apport report written because MaxReports is reached already
                                                              dpkg: dependency 
problems prevent processing triggers for libglib2.0-0:i386:
 libglib2.0-0:i386 depends on zlib1g (>= 1:1.2.2).

dpkg: error processing package libglib2.0-0:i386 (--configure):
 dependency problems - leaving triggers unprocessed
No apport report written because MaxReports is reached already
                                                              dpkg: dependency 
problems prevent processing triggers for libglib2.0-0:i386:
 libglib2.0-0:i386 depends on zlib1g (>= 1:1.2.2).

dpkg: error processing package libglib2.0-0:i386 (--configure):
 dependency problems - leaving triggers unprocessed
No apport report written because MaxReports is reached already
                                                              dpkg: dependency 
problems prevent processing triggers for libglib2.0-0:i386:
 libglib2.0-0:i386 depends on zlib1g (>= 1:1.2.2).

dpkg: error processing package libglib2.0-0:i386 (--configure):
 dependency problems - leaving triggers unprocessed
No apport report written because MaxReports is reached already
                                                              dpkg: dependency 
problems prevent processing triggers for libglib2.0-0:i386:
 libglib2.0-0:i386 depends on zlib1g (>= 1:1.2.2).

dpkg: error processing package libglib2.0-0:i386 (--configure):
 dependency problems - leaving triggers unprocessed
No apport report written because MaxReports is reached already
                                                              dpkg: dependency 
problems prevent processing triggers for libglib2.0-0:i386:
 libglib2.0-0:i386 depends on zlib1g (>= 1:1.2.2).

dpkg: error processing package libglib2.0-0:i386 (--configure):
 dependency problems - leaving triggers unprocessed
No apport report written because MaxReports is reached already
                                                              dpkg: dependency 
problems prevent processing triggers for libglib2.0-0:i386:
 libglib2.0-0:i386 depends on zlib1g (>= 1:1.2.2).

dpkg: error processing package libglib2.0-0:i386 (--configure):
 dependency problems - leaving triggers unprocessed
No apport report written because MaxReports is reached already
                                                              dpkg: dependency 
problems prevent processing triggers for libglib2.0-0:i386:
 libglib2.0-0:i386 depends on zlib1g (>= 1:1.2.2).

dpkg: error processing package libglib2.0-0:i386 (--configure):
 dependency problems - leaving triggers unprocessed
dpkg: too many errors, stopping
No apport report written because MaxReports is reached already
                                                              Errors were encou
ntered while processing:
 libglib2.0-0:i386
 libglib2.0-0:i386
 libglib2.0-0:i386
 libglib2.0-0:i386
 libglib2.0-0:i386
 libglib2.0-0:i386
 libglib2.0-0:i386
 libglib2.0-0:i386
 libglib2.0-0:i386
 libglib2.0-0:i386
 libglib2.0-0:i386
 libglib2.0-0:i386
 libglib2.0-0:i386
 libglib2.0-0:i386
 libglib2.0-0:i386
 libglib2.0-0:i386
 libglib2.0-0:i386
 libglib2.0-0:i386
 libglib2.0-0:i386
 libglib2.0-0:i386
 libglib2.0-0:i386
 libglib2.0-0:i386
 libglib2.0-0:i386
 libglib2.0-0:i386
 libglib2.0-0:i386
 libglib2.0-0:i386
 libglib2.0-0:i386
 libglib2.0-0:i386
 libglib2.0-0:i386
 libglib2.0-0:i386
 libglib2.0-0:i386
 libglib2.0-0:i386
 libglib2.0-0:i386
 libglib2.0-0:i386
 libglib2.0-0:i386
 libglib2.0-0:i386
 libglib2.0-0:i386
 libglib2.0-0:i386
 libglib2.0-0:i386
 libglib2.0-0:i386
 libglib2.0-0:i386
 libglib2.0-0:i386
 libglib2.0-0:i386
 libglib2.0-0:i386
 libglib2.0-0:i386
 libglib2.0-0:i386
 libglib2.0-0:i386
 libglib2.0-0:i386
 libglib2.0-0:i386
 libglib2.0-0:i386
Processing was halted because there were too many errors.
E: Sub-process /usr/bin/dpkg returned an error code (1)

**Steps to reproduce (if you know):**
go to pop!_shop and upgrade it

or write `sudo apt full-upgrade `

**Expected behaviour:**
It gets updated 

**Other Notes:**

**Comment by jacobgkau** [comment]
This seems like a potential duplicate of https://github.com/pop-os/pop/issues/1276, although I don't see the "not configured yet" portion of the error message. Since `zlib1g` appears to not be installed, please update your package lists, confirm that the candidate version is `>= 1:1.2.2`, and check why it's not able to be installed:

```
sudo apt update
apt policy zlib1g
sudo apt install zlib1g
```

---

System76 customers can reach out to [support](https://support.system76.com/) for technical assistance. For non-System76 hardware, you can seek community support on [Reddit](https://reddit.com/r/pop_os/) or [Mattermost](https://chat.pop-os.org).

**Comment by Amazeryogo** [comment]
When I do `apt policy zlib1g ` i get the following result:

zlib1g:
  Installed: 1:1.2.11.dfsg-2ubuntu1.2
  Candidate: 1:1.2.11.dfsg-2ubuntu1.2
  Version table:
 *** 1:1.2.11.dfsg-2ubuntu1.2 500
        500 http://us.archive.ubuntu.com/ubuntu focal-proposed/main amd64 Packages
        100 /var/lib/dpkg/status
     1:1.2.11.dfsg-2ubuntu1.1 500
        500 http://us.archive.ubuntu.com/ubuntu focal-updates/main amd64 Packages
     1:1.2.11.dfsg-2ubuntu1 500
        500 http://us.archive.ubuntu.com/ubuntu focal/main amd64 Packages

and when I try `sudo apt install zlib1g` i get this 

Reading package lists... Done
Building dependency tree       
Reading state information... Done
zlib1g is already the newest version (1:1.2.11.dfsg-2ubuntu1.2).
You might want to run 'apt --fix-broken install' to correct these.
The following packages have unmet dependencies:
 libavcodec58:i386 : Depends: zlib1g:i386 (>= 1:1.2.0) but it is not going to be installed
 libcairo2:i386 : Depends: zlib1g:i386 (>= 1:1.1.4) but it is not going to be installed
 libcups2:i386 : Depends: zlib1g:i386 (>= 1:1.2.0) but it is not going to be installed
 libcurl3-gnutls:i386 : Depends: zlib1g:i386 (>= 1:1.1.4) but it is not going to be installed
 libelf1:i386 : Depends: zlib1g:i386 (>= 1:1.1.4) but it is not going to be installed
 libfreetype6:i386 : Depends: zlib1g:i386 (>= 1:1.1.4) but it is not going to be installed
 libgd3:i386 : Depends: zlib1g:i386 (>= 1:1.1.4) but it is not going to be installed
 libgl1-mesa-dri:i386 : Depends: zlib1g:i386 (>= 1:1.1.4) but it is not going to be installed
 libglib2.0-0:i386 : Depends: zlib1g:i386 (>= 1:1.2.2) but it is not going to be installed
 libgstreamer-plugins-base1.0-0:i386 : Depends: zlib1g:i386 (>= 1:1.1.4) but it is not going to be installed
 libllvm10:i386 : Depends: zlib1g:i386 (>= 1:1.2.0) but it is not going to be installed
 libmysqlclient21:i386 : Depends: zlib1g:i386 (>= 1:1.1.4) but it is not going to be installed
 libosmesa6:i386 : Depends: zlib1g:i386 (>= 1:1.1.4) but it is not going to be installed
 libpci3:i386 : Depends: zlib1g:i386 (>= 1:1.1.4) but it is not going to be installed
 libpciaccess0:i386 : Depends: zlib1g:i386 (>= 1:1.2.3.3) but it is not going to be installed
 libperl5.30:i386 : Depends: zlib1g:i386 (>= 1:1.2.2.3) but it is not going to be installed
 libpng16-16:i386 : Depends: zlib1g:i386 (>= 1:1.2.11) but it is not going to be installed
 librtmp1:i386 : Depends: zlib1g:i386 (>= 1:1.1.4) but it is not going to be installed
 libssh-4:i386 : Depends: zlib1g:i386 (>= 1:1.1.4) but it is not going to be installed
 libtiff5:i386 : Depends: zlib1g:i386 (>= 1:1.1.4) but it is not going to be installed
 libwine:i386 : Depends: zlib1g:i386 (>= 1:1.1.4) but it is not going to be installed
                Recommends: libodbc1:i386 (>= 2.3.1) but it is not going to be installed
                Recommends: libasound2-plugins:i386 but it is not going to be installed
                Recommends: gstreamer1.0-plugins-good:i386 but it is not going to be installed
 libxml2:i386 : Depends: zlib1g:i386 (>= 1:1.2.3.3) but it is not going to be installed
 mesa-vdpau-drivers:i386 : Depends: zlib1g:i386 (>= 1:1.1.4) but it is not going to be installed
E: Unmet dependencies. Try 'apt --fix-broken install' with no packages (or specify a solution).

**Comment by Amazeryogo** [comment]
when I manually install it from https://packages.debian.org/jessie/zlib1g ,  and update it through eddy, and i get this 

The following packages have unmet dependencies:
  libgl1: Depends: libglx0 (= 1.3.1-1ubuntu0.20.04.1) but it is not going to be installed
  dictionaries-common: Depends: debconf (>= 1.5.5) but it is not going to be installed or
                                debconf-2.0
                       Depends: libtext-iconv-perl but it is not going to be installed
  libglx-mesa0: Depends: libgl1-mesa-dri but it is not going to be installed

**Comment by Amazeryogo** [comment]
so 😅  the pop!shop could barely work and the apt gave up and couldn't install anything, so 😅  I did a fresh install of  pop!_OS
